frontend/src/metabase/actions/containers/ActionCreator/ActionContext/ImplicitActionContextProvider/ImplicitActionContextProvider.tsx

```diff
@@ -33,12 +33,11 @@ function EditorBody() {
   );
 }
 
+// TODO: @uladzimir probably we can delete it, I'm not sure only can we get rid of getDefaultFormSettings
 function cleanFormSettings(formSettings?: ActionFormSettings) {
   const formSettingsWithDefaults = getDefaultFormSettings(formSettings);
 
-  // For implicit actions fields are generated dynamically according to the current schema
-  // For now, we don't let to customize the form to avoid dealing with fields getting out of sync
-  return _.omit(formSettingsWithDefaults, "fields");
+  return formSettingsWithDefaults;
```

Comment:
from now we do not remove `fields` and sync them with BE

Reply:
@kulyk could you please confirm if we sync settings with BE, we don't need to remove `fields` from settings? /cc @iethree 

(I'm tagging both of you as you worked on actions before)